cmd/agent/dist/conf.d/winkmem.d/conf.yaml.example

```diff
@@ -0,0 +1,29 @@
+init_config:
+  ## The Windows kernel memory check lists the top users of kernel memory by pool tag.
+  ## for each of the four metrics, configure the top N results to be reported.  The
+  ## default for all is 10.
+
+  ## topPagedBytes: N report the top N pool tag consumers of paged memory
```

Comment:
NIT: I believe we typically declare configurable parameters like this:
` ## @param topPagedBytes - integer - optional - default: 60`
(same for other cases below)

Reply:
I wouldn't consider that a nit ;)